packages/core/src/browser/keybinding.ts

```diff
@@ -293,12 +293,16 @@ export class KeybindingRegistry {
         /**
          * compare the given KeySequence with a particular binding
          */
-        function compareBinding(candidate: KeyCode[], binding: Keybinding) {
+        function compareBinding(candidate: KeyCode[], binding: Keybinding, isNormalized: boolean = false) {
             const bindingKeySequence = KeySequence.parse(binding.keybinding);
             const compareResult = KeySequence.compare(candidate, bindingKeySequence);
             switch (compareResult) {
                 case KeySequence.CompareResult.FULL: {
-                    result.full.push(binding);
+                    if (isNormalized) {
+                        result.normalized.push(binding);
```

Comment:
I don't understand, so we push normalized match on the side, but nothing picks it up?

Doesn't that breaks what the normalization is supposed to do?
(removing normalized full matches, and nobody picking it up?)